### PR TITLE
small adjust

### PR DESCRIPTION
**Pull request by retamiro**
Name user correction
Invalid sessionLogoff
Coverage 100%

### CODE REVIEW

File: src/application/modules/auth/useCases/logout/LogoutUseCase.test.ts [hunk 1]
```diff
@@ -1,22 +1,20 @@
-import { ILogProvider } from "../../../../shared/log/providerContracts/ILogProvider";
-import { ApplicationErrorMock } from "../../../../mocks/ApplicationError.mock";
 import applicationStatus from "../../../../shared/status/applicationStatus";
-import { LocaleTypeEnum } from "../../../../shared/locals/LocaleType.enum";
-import { UseCaseTraceMock } from "../../../../mocks/UseCaseTrace.mock";
-import { IAuthProvider } from "../../providerContracts/IAuth.provider";
-import { ISession } from "../../../../../domain/session/ISession";
-import { SessionMock } from "../../../../mocks/Session.mock";
 import appMessages from "../../../../shared/locals/messages";
```
Comment (harvic3) [inline src/application/modules/auth/useCases/logout/LogoutUseCase.test.ts:2]: Please organize imports from longest to shortest like before your changes.
I know, but code is art in some way, so we have to try to make it perfect.

File: src/adapters/providers/auth/Auth.provider.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export class AuthProvider extends BaseProvider implements IAuthProvider {
       this.userModel
         .getBySessionId(sessionId)
         .then((session) => resolve(!!session))
-        .catch(() => resolve(true));
+        .catch((err) => resolve(!!err));
```
Comment (harvic3) [inline src/adapters/providers/auth/Auth.provider.ts:52]: 👌

File: src/application/modules/auth/useCases/logout/LogoutUseCase.test.ts [hunk 3]
```diff
@@ -1,22 +1,20 @@
...
 import appWords from "../../../../shared/locals/words";
```
Comment (harvic3) [inline src/application/modules/auth/useCases/logout/LogoutUseCase.test.ts:3]: The same, imports from longest to shortest.

**Review by harvic3 — CHANGES_REQUESTED** [review]
Thanks for your passion @retamiro. 
Remember see and update your name like you would want in the CONTRIBUTIONS file.

File: src/application/modules/auth/useCases/logout/LogoutUseCase.test.ts (hunk 1 above)
Comment (retamiro) [reply src/application/modules/auth/useCases/logout/LogoutUseCase.test.ts:2]: It was a failure to understand the design pattern on my part. LOL
Correcting here

**Comment by retamiro** [comment]
@harvic3 

I thank you for doing a beautiful job on this project.

I have the ambition, in a good way, to contribute to expanding its use in the community, I'm going to take a look at "Tap Discussions", that's why I asked if you spoke Portuguese huahauhua

Done

**Review by harvic3 — APPROVED** [review]
Good job!! TK!! 🚀🙌

**Comment by harvic3** [comment]
@retamiro here is the discussion. 
https://github.com/harvic3/nodetskeleton/discussions/34

**Pull request merged**